examples/wasm-shell/components/wasm-terminal.tsx

```diff
@@ -65,14 +66,23 @@ const processWorkerUrl = (document.getElementById("worker") as HTMLImageElement)
 export default class WasmTerminalComponent extends Component {
   container: HTMLElement | null;
   wasmTerminal: WasmTerminal;
+  wasmFs: WasmFs;
 
   constructor() {
     super();
+    this.wasmFs = new WasmFs();
     this.wasmTerminal = new WasmTerminal({
       fetchCommand: fetchCommandHandler,
-      processWorkerUrl
+      processWorkerUrl,
+      wasmFs: this.wasmFs
     });
 
+    const TINY_PNG =
+      "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNkYPhfDwAChwGA60e6kgAAAABJRU5ErkJggg==";
+    const deser = new Buffer(TINY_PNG, "base64");
+    const contents = Uint8Array.from(deser);
+    this.wasmFs.volume.writeFileSync("/tiny.png", contents);
```

Comment:
This is just to assure that mounting binary files worked... which it did! (I also think is useful for the future for someone that wants to mount files).

Reply:
Yeah I am okay with this! 😄 👍